### ISSUE TITLE
Website doesn't work

### ISSUE THREAD

**Issue by davidhealey** [sfztools/sfizz#1308]
Looks like your domain expired or maybe the URL is wrong?? I'm just clicking the link in github.

**Comment by GeoffreyPlitt** [comment]
@redtide @paulfd Is this project having a crisis of ownership? Several us are willing to help if help is needed.

**Comment by atsushieno** [comment]
Looks like sfz.tools is dead. I believe what you are looking for now lives at: https://sfztools.github.io/

**Comment by tobiasBora** [comment]
One should fix the links, e.g. in the readme and description https://github.com/sfztools/sfizz

**Comment by forrcaho** [comment]
It's worth noting that `sfz.tools` has been squatted, and is serving out some really sketchy content. Fixing the links would help protect people from possible malware.

**Comment by pgcrowe** [comment]
Agreed. I would fix the links myself if I had access. One of the project contributors will have to do it. @atsushieno, @redtide, @paulfd?
